nix/lib/default.nix

```
@@ -1,8 +1,28 @@
-lib: {
-  dashToUnderscore =
-    attrset:
-    lib.mapAttrs' (name: value: {
-      name = lib.replaceStrings [ "-" ] [ "_" ] name;
-      inherit value;
-    }) attrset;
+lib: rec {
+  kebabToSnakeCase = lib.replaceStrings [ "-" ] [ "_" ];
```

```suggestion
  dashToUnderscore = lib.replaceStrings [ "-" ] [ "_" ];
  kebabToSnakeCase = dashToUnderscore;
```
Given that kebabToSnakeCase simply turns dashes into underscores, i feel this is a more appropriate name, with kebabToSnakeCase instead being an alias.

This relates to my next suggestion, where we reuse dashToUnderscore.